src/pages/home/Home.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useContext } from 'react'
+import { LoginContext } from '../../contexts/LoginContext'
+import { Link } from "react-router-dom";
+
+import "../../common.css"
+import "./home.css"
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faSignOutAlt } from '@fortawesome/free-solid-svg-icons'
+
+function LandingPage() {
+    // Imports the states using context from the parent component App.js
+    const { userName, setUserName, setPassword, setShowProfile }  = useContext(LoginContext);
+    const [bgColor, setBgColor] = useState("blue");
+
+    // Initials
+    let initials = "";
+    if(userName.split(" ").length > 1){
```

Comment:
When you add functions like this one, it makes sense to add a comment explaining its purpose.
Because I can see what it does, but it takes times to figure out.
Especially for people who are new to coding.

Reply:
not relevant anymore